api/src/main/java/org/openmrs/api/OrderService.java

```diff
@@ -627,6 +627,17 @@ public Order discontinueOrder(Order orderToDiscontinue, String reasonNonCoded, D
 	 */
 	@Authorized(PrivilegeConstants.GET_ORDER_TYPES)
 	public OrderType getOrderTypeByUuid(String uuid);
+
+	/**
+	 * Get OrderType by class  name
+	 *
+	 * @param className the class name used to get the order type
+	 * @since 2.4.0
```

Comment:
Shouldn't this be 2.5.0?

Reply:
@dkayiwa  don't we have Snapshot version of 2.5.0 currently?